charts/hdfs-datanode-k8s/README.md

```diff
@@ -6,10 +6,6 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
   Requires Kubernetes version 1.5 and beyond, because `namenode` is using
```

Comment:
Hmm. I see the same duplicate paragraph in the README.md of the namenode. Can we remove the existing paragraph and then put the new suggested paragraph in the top-level README.md, i.e. `charts/README.md`?